services/QuillLMS/client/app/bundles/Teacher/containers/PremiumPricingGuide.jsx

```diff
@@ -26,34 +29,99 @@ const subscribers = [
    { name: 'Princeton Public Schools logo', source: '/images/subscribers/16_princeton.png', id: 'princeton'}
  ]
 
-const PremiumPricingGuide = (props) => {
+export default class PremiumPricingGuide extends React.Component {
```

Comment:
I get that, since you were pulling logic out of the other component that was class-based, you decided to rewrite this as a class component, but since we are trying to move in the opposite direction in general it might have been a worthwhile effort to try to rewrite those functions in a hooks-based way.